ngm/api_views.py

```
@@ -192,8 +192,8 @@ def post(self, request):
     },
 )
 class CourtCaseDetailView(APIView):
-    authentication_classes = [TokenAuthentication]
-    permission_classes = [IsAuthenticated]
+    authentication_classes = []
```

Ruff (0.15.12)

[warning] Mutable default value for class attribute

(RUF012)